### ISSUE TITLE
Error: Error in migration engine. Reason: [libs/sql-schema-describer/src/walkers.rs:329:18] foreign key references unknown table: "Game" 

### ISSUE THREAD

**Issue by divyenduz** [prisma/prisma#4699]
Command: `prisma migrate save --experimental`
Version: `2.10.0`
Binary Version: `af1ae11a423edfb5d24092a85be11fa77c5e499c`
Report: https://prisma-errors.netlify.app/report/11820 
OS: `x64 darwin 19.6.0`

JS Stacktrace:
```
Error: Error in migration engine.
Reason: [libs/sql-schema-describer/src/walkers.rs:329:18] foreign key references unknown table: "Game"

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

    at ChildProcess.<anonymous> (/Users/larry-park/.config/yarn/global/node_modules/@prisma/cli/build/index.js:418:153)
    at ChildProcess.emit (events.js:314:20)
    at ChildProcess.EventEmitter.emit (domain.js:486:12)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
```

Rust Stacktrace:
```
[libs/sql-schema-describer/src/walkers.rs:329:18] foreign key references unknown table: "Game"
```

**Comment by janpio** [comment]
We have recent reports for this with 2.17

**Comment by tomhoule** [comment]
Yep, this is in the current milestone — I put more context into these crashes last sprint, if they keep happening we'll have some debugging help, because so far we don't know what is causing these.

**Comment by tomhoule** [comment]
Other error report: https://prisma-errors.netlify.app/report/13070

**Comment by tomhoule** [comment]
Let's monitor this, but I am pretty sure this is fixed in 2.18.0 — if we see no new reports with 2.18.0, we can close.

**Comment by tomhoule** [comment]
Discussed with @do4gr and it doesn't happen _only_ in these circumstances, so we'll do the same fix as on sqlite here. PR coming.
